src/commons/__mocks__/logger.ts

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
What effect is this piece of code having? I thought using `jest.mock("../../commons/logger");` is enough? Interestingly, removing this file shows that all the tests pass, which suggests this file isn't needed?